### PR TITLE
Update Democratic Republic of The Congo

### PR DESCRIPTION
**Pull request by mgrassotti**
Trello: https://trello.com/c/HOnUP71d/493-democratic-republic-of-the-congo-fixes-medium

Update the name of the Democratic Republic of The Congo

### CODE REVIEW

**Comment by Davidslv** [comment]
Don't change the slug, just the name.

This looks like a follow up of: https://trello.com/c/rxjklHjL/191-country-name-change-democratic-republic-of-the-congo

File: lib/data/countries.yml [hunk 1]
```diff
@@ -111,8 +111,8 @@
   slug: cyprus
 - name: Czech Republic
   slug: czech-republic
-- name: Democratic Republic of Congo
-  slug: democratic-republic-of-congo
+- name: Democratic Republic of The Congo
+  slug: democratic-republic-of-the-congo
```
Comment (Davidslv) [inline lib/data/countries.yml]: the slug should be kept as it was.

**Comment by tijmenb** [comment]
@Davidslv You need to provide some more context here. This is a different app from rummager & finder-frontend, so changing the slug here could very well be the right thing to do.

**Comment by Davidslv** [comment]
@mgrassotti, I was wrong, I was thinking it would conflict with rummager functionality as @tijmenb mentioned.
Sorry about that.

**Pull request merged**